test/test-assertions.rb

```diff
@@ -2120,12 +2120,11 @@ def test_fail_with_message
       end
 
       def test_fail_with_float_like_object
-        x = BigDecimal("0.1")
         object = Object.new
         def object.to_f
           0.4
         end
-        inspected_object = AssertionMessage.convert(object)
+        AssertionMessage.convert(object)
```

Comment:
Could you use `inspected_object` in the `check_fail()` argument instead of removing `inspected_object`?